app/src/main/java/com/example/pocketcloset/RandomizedView.java

```diff
@@ -0,0 +1,124 @@
+package com.example.pocketcloset;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+import com.example.pocketcloset.adapters.ClothingAdapter;
+import com.example.pocketcloset.models.Clothing;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomizedView extends AppCompatActivity {
+    protected List<Clothing> allClothing;
+    private ImageView ivShoes;
+    private ImageView ivTop;
+    private ImageView ivBottom;
+    private ImageView ivBracelet;
+    private ImageView ivHeadwear;
+    private ImageView ivNeckwear;
+    private ImageView ivHandheld;
+    private ClothingAdapter adapter;
+    private Button btnRandomize2;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_randomized_view);
+
+
+        ivShoes = findViewById(R.id.ivShoes);
+        ivTop = findViewById(R.id.ivTop);
+        ivBottom = findViewById(R.id.ivBottom);
+        btnRandomize2 = findViewById(R.id.btnRandomize2);
+        ivHeadwear = findViewById(R.id.ivHeadwear);
+        allClothing = new ArrayList<>();
+        ivNeckwear = findViewById(R.id.ivNecklace);
+        adapter = new ClothingAdapter(this, allClothing);
+        queryClothes(null);
+        Log.i("FRAGMENT CLOTHES", "b" + allClothing.size());
```

Comment:
There are various opinions on what logging should or should not be committed. I don't have a particularly strong opinion here, but if you're committing it, consider improving the log message. We all take shortcuts like `"b"` when developing on the inner loop, but long term the message won't make much sense. Suggest something like `"Total Clothing Items: "`.